client/components/UI/List/PubStatus.tsx

```diff
@@ -2,24 +2,26 @@ import React from 'react';
 import {OverlayTrigger, Tooltip} from 'react-bootstrap';
 import PropTypes from 'prop-types';
 
+import {superdeskApi} from '../../../superdeskApi';
+
 import {Column} from './Column';
-import {isNotForPublication, gettext} from '../utils';
-import {TOOLTIPS} from '../constants';
+import {isNotForPublication} from '../utils';
 
 /**
  * @ngdoc react
  * @name PubStatus
  * @description Component to show published status of an item
  */
 export const PubStatus = ({item, isPublic}) => {
+    const {gettext} = superdeskApi.localization;
     let badge;
     let title = null;
 
     if (isPublic) {
-        title = gettext(TOOLTIPS.postedState);
+        title = gettext('Posted');
```

Comment:
why not use the tooltips here? seems like those are translated

Reply:
That was a change from my first pass, before I got the Tooltips. I'll update that